templates/dashboard/product/product_variant/form.html

```diff
@@ -76,10 +76,20 @@
                 <div class="row">
                   {{ form.name|materializecss }}
                 </div>
+                <p class="card-title">{% trans "Stock" %}</p>
```

Comment:
What about:
- moving this section under the `SKU` field,
- renaming the label "quantity" to "stock quantity",
- and renaming the label "price override" to "selling price"?

Reply:
Moving stock to the right doesn't look good when variant has no attributes, will think about this UI further later

Reply:
Okay then, sounds fair!

> - renaming the label "quantity" to "stock quantity",
> - and renaming the label "price override" to "selling price override" (and thus "Selling price on base product form label)

I think those two could be great to add as they would be more descriptive.

Reply:
@Pacu2 I don't think we want to support variants with no attributes.

Reply:
Take a look at the new screenshot

Reply:
For new screenshot:

* I don't think "number in stock" belongs in the wide section as it we are unlikely to deal with stock in excess of ten digits 😉 
* sale price and cost price are different width
* sale price has double height which looks kind of odd (in other forms only the first line of the form behaves this way)

Reply:
Screenshot updated according to @patrys comments